modules/distr-tracing-tempo-install-cli.adoc

```diff
@@ -0,0 +1,231 @@
+// Module included in the following assemblies:
+//
+//* distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_content-type: PROCEDURE
+[id="distr-tracing-tempo-install-cli_{context}"]
+= Installing the {TempoShortName} by using the CLI
+
+You can install the {TempoShortName} from the command line.
+
+.Prerequisites
+
+* An active OpenShift CLI (`oc`) session by a cluster administrator with the `cluster-admin` role.
++
+[TIP]
+====
+* Ensure that your OpenShift CLI (`oc`) version is up to date and matches your {product-title} version.
+
+* Run `oc login`:
++
+[source,terminal]
+----
+$ oc login --username=<your_username>
+----
+====
+
+* You are using a supported provider of object storage: link:https://www.redhat.com/en/technologies/cloud-computing/openshift-data-foundation[Red Hat OpenShift Data Foundation], link:https://min.io/[MinIO], link:https://aws.amazon.com/s3/[Amazon S3], link:https://azure.microsoft.com/en-us/products/storage/blobs/[Azure Blob Storage], link:https://cloud.google.com/storage/[Google Cloud Storage].
+
+.Procedure
+
+. Install the {TempoOperator}:
+
+.. Create an OpenShift project for the {TempoOperator}.
++
+[source,terminal]
+----
+$ oc apply -f - << EOF
+apiVersion: project.openshift.io/v1
+kind: Project
+metadata:
+  labels:
+    kubernetes.io/metadata.name: openshift-tempo-operator
+    openshift.io/cluster-monitoring: "true"
+  name: openshift-tempo-operator
+EOF
+----
+
+.. Create an Operator group.
++
+[source,terminal]
+----
+$ oc apply -f - << EOF
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: openshift-tempo-operator
+  namespace: openshift-tempo-operator
+spec:
+  upgradeStrategy: Default
+EOF
+----
+
+.. Create a subscription.
++
+[source,terminal]
+----
+$ oc apply -f - << EOF
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: tempo-product
+  namespace: openshift-tempo-operator
+spec:
+  channel: stable
+  installPlanApproval: Automatic
+  name: tempo-product
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+EOF
+----
+
+.. Check the Operator status.
++
+[source,terminal]
+----
+$ oc get csv -n openshift-tempo-operator
+----
+
+. Run `oc apply -f` for a YAML file or standard input to create a Secret for your object storage bucket:
++
+[source,terminal]
+----
+$ oc apply -f <secret_file>.yaml
+----
++
+[source,terminal]
+----
+$ oc apply -f - << EOF
+<object_storage_secret>
+EOF
+----
```

Comment:
Is "Secret" an API object? Typically we use either "secret" or "`Secret` object", rather than treating it as a plain proper noun.

Command steps need "by running the following command:" at the end. Also cannot put two commands in one step. Possible approach:

```suggestion
. Create a secret for your object storage bucket by running one of the following commands:

** To create a secret from a YAML file:
+
[source,terminal]
----
$ oc apply -f <secret_file>.yaml
----

 ** To create a secret from standard input:
+
[source,terminal]
----
$ oc apply -f - << EOF
<object_storage_secret>
EOF
----
```

Reply:
Fantastic suggestion! Thank you. Applying manually.